cmd/exo/cmd/vm_stop.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+// stopCmd represents the stop command
+var vmStopCmd = &cobra.Command{
+	Use:   "stop <vm name> [vm name] ...",
+	Short: "Stop virtual machine instance",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return cmd.Usage()
+		}
+
+		for _, v := range args {
+			if err := asyncStopWithCtx(gContext, v); err != nil {
+				fmt.Fprintln(os.Stderr, err) // nolint: errcheck
+			} else {
+				fmt.Println("\nStopped !")
+			}
+		}
+		return nil
```

Comment:
I don't like this `return nil`

Reply:
I dont want to stop all args vm stopping so I just display error on stderr and continue the flow.
that why I return nil at the end

Reply:
gather all the errors and return one of them in case of a failure.